program/src/instruction.rs

```diff
@@ -69,11 +70,9 @@ pub enum LidoInstruction {
     /// Observe any external changes in the balances of a validator's stake accounts.
     ///
     /// If there is inactive balance in stake accounts, withdraw this back to the reserve.
+    /// Distribute fees.
     WithdrawInactiveStake,
 
-    /// Claim rewards from the validator account and distribute rewards.
-    CollectValidatorFee,
-    ClaimValidatorFee,
```

Comment:
We can’t remove elements from this enum, because it would change the serialization format. A Lido instruction starts with a discriminant byte that indicates the instruction; removing these two would change that byte for all instructions that follow here, which would break everything that submits transactions.

Instead, we should mark them as deprecated. We can replace the handlers for these instructions with no-op handlers that make the transaction fail with a message like “this instruction is no longer supported”. But we need to keep them around in the enum forever.